app/controllers/synthetic_category_controller.rb

```diff
@@ -0,0 +1,47 @@
+# Only a 'show' action
+#
+# Based roughly off of Sufia/CurationConcerns CollectionController#show
+# TODO link to CollectionControllerBehavior in both
+class SyntheticCategoryController < ApplicationController
+    # include Blacklight::AccessControls::Catalog
+    include Blacklight::Base
+
+    # ??
+    copy_blacklight_config_from(::CatalogController)
+
+    def show
+      unless synthetic_category
+        raise ActionController::RoutingError.new("No SyntheticCategory matches `#{params[:id]}`")
+      end
+      set_response # trigger setting of @response, which I think some partials need.
+    end
+
+
+    protected
+
+    # Get controller to find templates in CollectionController too,
+    # so we can re-use more of the sufia stuff for collection. Messy messy.
+    def self.local_prefixes
```

Comment:
I guess a future version of Rails might break it. I'm not sure how to test for it without kind of copying Rails tests. 

Hmm, I guess I could write an integration/capybara test for this synthetic category page. I guess that'd be the right way to do it, that'd fail if the view template lookup stuff broke. And might be the answer to your question above "I am wondering if there are tests it would be useful to add here". 

What do you think, is it finally time to break through our integration test reluctance, and get one written for this, as a pre-req to it getting merged?  I'm not really sure.